.eslintrc.json

```diff
@@ -59,8 +59,25 @@
 		"comma-dangle": "error",
 		"keyword-spacing": "error",
 		"space-in-parens": ["error", "never"],
-		"func-style" : ["warn", "expression"],
+		"func-style" : ["error", "expression"],
 		"require-await": "off",
-		"@typescript-eslint/no-floating-promises": ["error"]
-	}
+		"@typescript-eslint/no-floating-promises": ["error"],
+		"jest/expect-expect": [
+			"error",
+			{
+				"assertFunctionNames": [
+					"expect",
+					"supertest.**.expect",
+					"*.expect"
+				]
+			}
+		]
+	},
+	"overrides": [{
+		"files": [ "src/**/*.test.ts" ],
+		"rules": {
+			"@typescript-eslint/no-explicit-any": "off",
+			"@typescript-eslint/no-non-null-assertion": "off"
```

Comment:
This is for making `DatabaseStateCreator.gitHubServerApp!.gitHubAppId` pass eslint.
We do want the `!` after the `gitHubServerApp` for fail-fast. So I think it is better to allow it in tests files.

In runtime, allowing `!` may cause a bug, while in test time, allow `!` should fail-fast (which is better).